ui/scss/component/_file-price.scss

```diff
@@ -22,15 +22,6 @@ $SKEW_AMOUNT: 10deg;
   }
 }
 
-.filePrice--skewedBg {
-  transform: skew($SKEW_AMOUNT);
-
-  .icon,
-  .credit-amount {
-    transform: skew(-$SKEW_AMOUNT);
-  }
-}
```

Comment:
There is one more `.filePrice--filepage.filePrice--skewedBg ` to cleanup if no longer used, rather than left there?